config.json

```diff
@@ -0,0 +1 @@
+{"api-token":"303d3b3541565530fb3836f080d6e7c173dbaab7","email":"shmuel@practitest.com","project-id":2,"additional-test-fields":{"custom-fields":{"---f-7":"?compiler"},"system-fields":{}},"additional-testset-fields":{"custom-fields":{},"system-fields":{}}}
```

Comment:
Please do not commit api tokens to public repository!

Reply:
done (local token)